ui/src/dataLoaders/components/DataLoadersOverlay.scss

```diff
@@ -41,4 +41,60 @@
   border-radius: 0 $radius $radius 0;
   flex: 1 0 auto;
   padding: $ix-marg-d;
-}
\ No newline at end of file
+}
+
+/*
+  Telegraf Instructions Overlay
+  ------------------------------------------------------------------------------
+*/
+
+.telegraf-instructions {
+  p, h6 {
+    font-size: 14px;
+  }
+
+  p {
+    margin-bottom: 2em;
+  }
+}
+
+/*
+  Line Protocol Wizard
+  ------------------------------------------------------------------------------
+*/
+
+.line-protocol--header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: $ix-marg-b;
+  z-index: 5050;
+  position: relative;
+}
+
+.line-protocol--content {
+  height: 300px;
+}
+
+.line-protocol--status {
+  font-size: 13px;
+  font-weight: 600;
+  user-select: none;
+
+  &.done {
+    color: $c-rainforest;
+  }
+  &.error {
+    color: $c-dreamsicle;
+  }
+}
+
+.line-protocol--spinner {
+  height: 300px + $ix-marg-b + 30px;
```

Comment:
you only need `calc()` if you are combining a `%` with a `px` or other unit

I should probably make the 300 into a variable so that the 2 places it is used are connected

Reply:
Good to know, thanks!